bower.json

```diff
@@ -18,7 +18,9 @@
     "angular-emoji": "angular-emoji-filter#~0.0.2",
     "angular-sanitize": "~1.5.0",
     "json-formatter": "~0.5.0",
-    "velocity": "~1.2.3"
+    "velocity": "~1.2.3",
+    "ngclipboard": "~1.1.1",
+    "ng-device-detector": "~3.0.1"
```

Comment:
I left this here even though I didn't use it, since we'll need this for MixPanel